chandra_aca/centroid_resid.py

```diff
@@ -130,10 +130,12 @@ def set_atts(self, source):
             atts = np.vstack([telem['aoattqt{}'.format(idx)].vals
                               for idx in [1, 2, 3, 4]]).transpose()
             att_times = telem['aoattqt1'].times
-            obsids = fetch.Msid('COBSRQID', tstart, tstop)
-            if np.unique(obsids.vals) > 1:
+            # Fetch COBSQID at beginning and end of interval, check they match, and defined obsid
+            obsid_start = fetch.Msid('COBSRQID', tstart, tstart + 60)
+            obsid_stop = fetch.Msid('COBSRQID', tstop - 60, tstop)
+            if obsid_start.vals[-1] != obsid_stop.vals[0]:
```

Comment:
Realistically there will never be support for a time range that spans obsids.  Just change to `ValueError` and don't hint at support in the future.  Also (following newer astropy and Python protocol), exception messages should always start with lower case.